templating/twig_extension.rst

```diff
@@ -15,8 +15,14 @@ your code faster.
 
 .. tip::
 
-    Before writing your own extensions, have a look at the
-    `Twig official extension repository`_.
+    When writing your own extensions, you might want to learn
+    from having a look at the `Twig Bridge`_  which contains most of
+    the extensions provided by the Symfony Framework.
+
+.. tip::
+
+    We also have :doc:`a short article </templating/twig_extension_repository>`
```

Comment:
We always avoid the first person perspective in the docs. This could be rewritten to something like "Read ... on how to use [...]"